tailwind.config.js

```diff
@@ -36,6 +36,9 @@ module.exports = {
           500: "#b7e0ff",
         },
       },
+      width: {
+        "90": "90%"
```

Comment:
I don't think this is needed, when using custom size for width, you can do `w-[90%]`

Reply:
done :3